libsplinter/src/auth/rest_api/mod.rs

```diff
@@ -56,6 +56,11 @@ fn authorize(
     if endpoint.starts_with("/oauth") {
         is_auth_endpoint = true;
     }
+    #[cfg(feature = "oauth")]
+    if endpoint == "/oauth/logout" {
+        is_auth_endpoint = false;
+    }
+
```

Comment:
Can you change this to just set the bool to `true` if the endpoint is `/oauth/login` or `/oauth/callback`, like the biome endpoints?